django_redis/client/default.py

```diff
@@ -315,6 +318,9 @@ def pexpire(
         version: Optional[int] = None,
         client: Optional[Redis] = None,
     ) -> bool:
+        if (timeout is DEFAULT_TIMEOUT) or (timeout is None):
```

Comment:
I would let people set None as timeout and get the exception from the client so that they notice they made a mistake instead of getting a seemingly random expiry time

Reply:
Thanks for your feedback @WisdomPill!
Changes are done. Please approve the workflow.